README.md

```diff
@@ -1,11 +1,16 @@
 [![Build Status](https://dev.azure.com/Lumispy/lumispy/_apis/build/status/LumiSpy.lumispy?branchName=master)](https://dev.azure.com/Lumispy/lumispy/_build/latest?definitionId=3&branchName=master)
 ![Tests](https://github.com/lumispy/lumispy/workflows/Tests/badge.svg)
+[![Build status](https://github.com/pyxem/kikuchipy/workflows/build/badge.svg)](https://github.com/pyxem/kikuchipy/actions)
```

Comment:
It seems that the copy and paste is missing some update! :)
This badge is the same as the "Tests" badge and the previous one is missing the link to the actions tab..

Reply:
Indeed, and the workflow is actually called tests in our case and already included. I was a bit quick on that one.